src/lib/history.test.ts

```diff
@@ -0,0 +1,55 @@
+import { getMondayFromDate, getArticleViewCountForWeeks } from './history';
+
+const oneDay = 86400000;
+
+describe('getMondayFromDate', () => {
+    it('should return Monday midnight if today is Monday', () => {
+        const mondayMorning = new Date('2020-03-02T09:15:30');
+        const mondayMidnight = Math.floor(mondayMorning.getTime() / oneDay);
```

Comment:
I wonder if we could create our expected `mondayMidnight` here explicitly, instead of deriving it from other values? So something like:

```js
const mondayMidnight = new Date('2020-03-02T00:00:00').getTime();
```

That would remove the intermediate `mondayMorning` const, which I think could benefit readability. What do you reckon?

Reply:
I've refactored to declare var as simple Date object, then convert to the expected format when comparing with output of the function being tested.